docs/content/envoy-primer.md

```diff
@@ -145,12 +187,32 @@ Sample input received by OPA is shown below:
 }
 ```
 
```

Comment:
Nit: We should update the token in the above input if we want a `200` status. That token is expired.